SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -23,48 +23,38 @@ _log () {
     fi
```

Comment:
The commit message for  swancustomenvironments: Remove repo_type  has typos, please fix.